metadata-models/src/main/pegasus/com/linkedin/chart/ChartInfo.pdl

```diff
@@ -0,0 +1,46 @@
+namespace com.linkedin.chart
+
+import com.linkedin.common.AccessLevel
+import com.linkedin.common.ChangeAuditStamps
+import com.linkedin.common.Time
+
+/**
+ * Information about a chart
+ */
+record ChartInfo {
+
+  /**
+   * Title of the chart
+   */
+  title: string
+
+  /**
+   * Detailed description about the chart
+   */
+  description: string
+
+  /**
+   * Captures information about who created/last modified/deleted this chart and when
+   */
+  lastModified: ChangeAuditStamps
```

Comment:
why do we need this field? aspects are stored with the corresponding timestamp in the table?

Reply:
`ChangeAuditStamps` is composed of 3 `AuditStamp` actually. It stores creation, last modification and deletion AuditStamp separately. AuditStamp itself is not enough if you wanna capture who/when created the chart.